### DISCUSSION TITLE
The Mod Bounty Board!

### DISCUSSION THREAD

**Discussion by metamancer** [metamancer/frostytoolsuite#1] [Announcements]
### Mod Requests
This discussion area can help members of the community request mods or look up modders available to take requests.  [Post your mod request](https://github.com/metamancer/DAVModdersResource/discussions/3) and leave it on the bounty board for someone to pick up!

### Modders
If you make mods for others, you can [post a profile](https://github.com/metamancer/DAVModdersResource/discussions/2) of your own and tell people about the work you do, share commission information and showcase some of your work. 

### Artists, Writers and Other Creators
If you don't make mods specifically, or do but also are interested in using your skillset to contribute to a modding project, you can also [post a profile](https://github.com/metamancer/DAVModdersResource/discussions/2)! 

## Posting Guidelines
Please follow the appropriate formatting when posting and remember: 
  * A request does not equal demand or entitlement. No one is under any obligation to take on a mod request.
  * Collaboration is better than competition.
  * Welcome others and be kind. This is a community we build together 💪.

**Comment by Cullenite** [comment]
I really want to swap Davrin and Emmrich to look like Lucanis, so I can have those romance scenes. But I know Davrin is taller than Lucanis, so that would make the animations a bit off. So, I would like a swap, but an edit to Lucanis' height to be the same as Davrin and another mod that does the same, but with Emmrich. If anyone could help, I would be so grateful!

**Reply by metamancer** [reply]
Hey there are already quite a few mods that change Lucanis, as well as swap. For example this [Taller Lucanis](https://www.nexusmods.com/dragonagetheveilguard/mods/1341) mod. I recommend checking on nexus that there aren't already mods which help you reach this goal. I'm assuming based on the start of your post you're going to be doing the swaps yourself? If so, I think that mod linked may help with your height request. Good luck!

**Reply by Cullenite** [reply]
I've checked any swaps for Davrin to Lucanis, but I can't find any. I've seen plenty of swaps, but not that particular one. Whenever I try to do it myself, Davrin is either ending up with no head but keeps his regular body, or just completely invisible. And I launch the game via the Editor, so no other mods are in use to conflict.

**Reply by metamancer** [reply]
If you're requesting a new mod then be sure to complete the request template and start a new discussion for it in the mod request board. Directions are at the top of this page and it points to the template as well:
https://github.com/metamancer/DAVModdersResource/discussions/3
